start.js

```diff
@@ -11,16 +11,19 @@ if (process.env.NODE_ENV === 'dev') {
 
 const express = require('express')
 const path = require('path');
+const morgan = require('morgan')
```

Comment:
[nit] are we using semicolons or nah

Reply:
uhhh I think no semicolons

I kind of automatically add semicolons, but this article explains why it's not that useful to: https://feross.org/never-use-semicolons/

Good catch!

Reply:
We should add a linter to catch/auto-fix these small things